packages/graphback/src/ContextCreator.ts

```diff
@@ -15,12 +15,35 @@ export const createInputContext = (schemaText: string, defaultConfig: Config): T
 
     const schemaDef = visit(astNode, { leave: inputTypeVisitor });
 
-    return schemaDef.definitions.map((t: Type) => {
+    let context =  schemaDef.definitions.map((t: Type) => {
       return {
         ...t,
         config: { ...defaultConfig, ...t.config }
       }
     })
+
+
+
+    const extendNodes = context.filter((t: Type) => t.kind === OBJECT_TYPE_EXTENSION)
```

Comment:
You could use the available util function to do this filter.

https://github.com/aerogear/graphback/blob/2aeaeac9d57c2fdcc6aaf5cc8baeb698d548d766/packages/graphback/src/utils/index.ts#L21